contentcuration/contentcuration/views.py

```diff
@@ -354,8 +354,10 @@ def _duplicate_node(node, sort_order=1, parent=None, channel_id=None):
         aiobj_copy.contentnode = new_node
         aiobj_copy.save()
 
+    sort_order = 1
     for c in node.children.all():
-        _duplicate_node(c, parent=new_node.id)
+        _duplicate_node(c, sort_order=sort_order, parent=new_node.id)
```

Comment:
Why would it not just use the sort_order value from the original node? I guess it depends on whether it's being cloned recursively (in which case it could be the same) or a top-level node going into a topic that already has other topics (in which case it should be like this but start counting from the last sort_order of the existing siblings.

Perhaps let's chat through it in person.